subsys/bluetooth/services/fast_pair/fp_storage/Kconfig.fp_storage

```diff
@@ -30,6 +30,30 @@ config BT_FAST_PAIR_STORAGE_EXPOSE_PRIV_API
 	  Expose Fast Pair storage's private data structures, definitions and functionalities.
 	  The option could be used by unit tests to prepopulate settings with mocked data.
 
+config BT_FAST_PAIR_STORAGE_EXT_PN
+	bool "Fast Pair storage for Personalized Name extension"
+	help
+	  Add Fast Pair storage source files for Personalized Name extension.
+
+if BT_FAST_PAIR_STORAGE_EXT_PN
+
+config BT_FAST_PAIR_STORAGE_EXT_PN_LEN_MAX
+	int "Maximum length of stored Fast Pair Personalized Name"
+	range 64 200
```

Comment:
I wonder if we could increase the upper bound to avoid introducing artificial limitation to our users (maybe we could even make it e.g. 1024)?